### PR TITLE
Add descriptions for commands

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Make the 1st column expand instead of the 0th one so the description always takes the most space

image

### CODE REVIEW

**Comment by AnidemDex** [comment]
Epic embed fail

**Comment by Crystalwarrior** [comment]
☠

File: commands/command_animate.gd [hunk 1]
```diff
@@ -8,5 +8,9 @@ func _get_command_name() -> String:
 	return "Animate"
 
 
+func _get_command_desc() -> String:
```
Comment (AnidemDex) [inline commands/command_animate.gd:11]: No needed

File: commands/command_call.gd [hunk 2]
```diff
@@ -8,5 +8,9 @@ func _get_command_name() -> String:
 	return "Call"
 
 
+func _get_command_desc() -> String:
```
Comment (AnidemDex) [inline commands/command_call.gd:11]: Description here is only needed if there's something to show, not as placeholder

**Review by AnidemDex — COMMENTED** [review]
We should add proper placeholder for commands. For now let's just leave them blank if it's not related.

File: commands/command_animate.gd (hunk 1 above)
Comment (Crystalwarrior) [reply commands/command_animate.gd:11]: ?

**Review by AnidemDex — APPROVED** [review]
All fine

**Pull request merged**